search-commons/src/main/java/no/unit/nva/search2/common/Query.java

```diff
@@ -105,21 +104,6 @@ public PagedSearch toPagedResponse(SwsResponse response) {
                 .build();
     }
 
-    /**
-     * Builds URI to query SWS based on post body.
-     *
-     * @return an URI to Sws search without parameters.
-     */
-    public URI getOpenSearchUri() {
```

Comment:
This is the adresse to opensearch index and must be defined for each Query instance